docs/course/course.yaml

```
@@ -26,83 +26,106 @@ paths:
+                              name: Languages
+                            _id: 63ed1cd25e9d781cdb6a6b17
                             resourceType: lesson
-                          },
-                          {
-                            resource:
-                              {
-                                _id: 64ba39c714b7bc037a29d712,
-                                title: Title,
-                                description: description,
-                                items: [6477007a6fa4d05e1a800ce4, 6477007a6fa4d05e1a800ce6],
-                                author: '6477007a6fa4d05e1a800ce5',
-                                category: '63bed9ef260f18d04ab15da',
-                                settings:
-                                  {
-                                    view: Scroll,
-                                    shuffle: true,
-                                    pointValues: false,
-                                    scoredResponses: false,
-                                    correctAnswers: true
-                                  }
-                              },
-                            resourceType: quiz
-                          },
-                          {
-                            resource:
-                              {
-                                _id: 63ed1cd25e9d781cdb6a6b15,
-                                author: 648afee884936e09a37deaaa,
-                                fileName: EnglishB2Level,
-                                link: 'https://english-b2-test.com',
-                                size: 230,
-                                category: { _id: 6477007a6fa4d05e1a800ce1, name: Science }
-                              },
+                            createdAt: 2023-12-01T13:25:36.292Z
+                            updatedAt: 2023-12-01T13:25:36.292Z
+                          resourceType: lesson
+                        - resource:
+                            _id: 64ba39c714b7bc037a29d712
+                            title: Title
+                            description: description
+                            items:
+                              - 6477007a6fa4d05e1a800ce4
+                              - 6477007a6fa4d05e1a800ce6
+                            author: 6477007a6fa4d05e1a800ce5
+                            category: 63bed9ef260f18d04ab15da
+                            settings:
+                              view: Scroll
+                              shuffle: true
+                              pointValues: false
+                              scoredResponses: false
+                              correctAnswers: true
+                          resourceType: quiz
+                        - resource:
+                            _id: 63ed1cd25e9d781cdb6a6b15
+                            author: 648afee884936e09a37deaaa
+                            fileName: EnglishB2Level
+                            link: 'https://english-b2-test.com'
```

Check, please, if it works without quotes
```suggestion
                            link: https://english-b2-test.com
```